releases.json

```diff
@@ -39,11 +39,12 @@
     "ignore_vulnerabilities": [],
     "releases": [
         {
-            "device": "inf2",
-            "version": "0.0.21",
+            "device": "gpu",
```

Comment:
sanity-check: Is this expected?

Reply:
yeah it's expected. inf2 is https://aws.amazon.com/ec2/instance-types/inf2/ more powerful than normal gpu instances.